.gitignore

```diff
@@ -34,3 +34,6 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+# firebase
+.firebase
```

Comment:
New line after `.firebase`

Reply:
just add space?

Reply:
<img width="198" alt="Screenshot 2023-09-28 at 12 17 14 PM" src="https://github.com/penumbra-zone/web/assets/16624263/a5ff2721-e70c-4d3c-9840-bff797ca02a8">

Yes, add a space below. You can see github giving a warning when you don't.